server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -490,7 +490,7 @@ private static long jitter(long ms) {
         return watcher.run(Constants.BULK_ARBITRATOR_TYPE, tid, () -> {
           List<TKeyExtent> failures = new ArrayList<>();
 
-          for (Entry<TKeyExtent,Map<String,MapFileInfo>> entry : files.entrySet()) {
+          files.entrySet().forEach((entry) -> {
```

Comment:
`files.forEach((k,v) -> {})` would be cleaner than `files.entrySet().forEach(entry -> { entry.getKey()... entry.getValue()...})`.

Reply:
Thanks.  Didn't know that would work.  Do you want me to keep working on this?

Reply:
I made that correction @ctubbsii.  Found a lot of other loops to do the same thing.  Finished with the TabletServer.java file.  A couple of other loops are in there but they had counting variables in them which needed to be declared final and I didn't want to risk breaking the code.

Reply:
I think some of these probably do read better with the lambda (especially the ones that loop over maps, where we can remove the intermediate `Entry` object in favor of `(k, v)`), but I'm not sure that's the case for all of them. I'll leave it to you to decide if it's worth your time to change them.

Keep in mind that some of them probably can't be changed, because they throw checked exceptions, and I don't think that's allowed on the lambda form.